### PR TITLE
Scip bump

### PR DESCRIPTION
**Pull request by matbesancon**
now adding GMP as an explicit dependency since SCIP adds it anyway

### CODE REVIEW

**Comment by matbesancon** [comment]
ping @rschwarz

File: S/SCIP/build_tarballs.jl [hunk 1]
```diff
@@ -49,7 +49,7 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     Dependency(PackageSpec(name="bliss_jll", uuid="508c9074-7a14-5c94-9582-3d4bc1871065"))
-    Dependency(PackageSpec(name="Ipopt_jll", uuid="9cc047cb-c261-5740-88fc-0cf96f7bdcc7"))
+    Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d"))
```
Comment (giordano) [inline S/SCIP/build_tarballs.jl:52]:
```suggestion
    Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d"), v"6.1.2")
```

**Comment by giordano** [comment]
You need also to rebase on master

**Comment by rschwarz** [comment]
Doesn't this require another version bump?

**Comment by matbesancon** [comment]
I would say no, we just add a dependency that was necessary, so a +1 after the patch will be sufficient

**Pull request merged**